csunplugged/resources/utils/resize_encode_resource_images.py

```diff
@@ -0,0 +1,41 @@
+from PIL import Image
+from io import BytesIO
+import base64
+
+MM_TO_PIXEL_RATIO = 6
+
+
+def resize_encode_resource_images(generator, data):
```

Comment:
Probably better to only pass through the paper size here, not the whole generator object, to reduce coupling.